targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/PeripheralNames.h

```diff
@@ -124,14 +124,19 @@ typedef enum {
     DAC_0 = 0
 } DACName;
 
-
 #define SPI_COUNT 3
 typedef enum {
     SPI_0 = 0,
     SPI_1 = 1,
     SPI_2 = 2,
 } SPIName;
 
+/* Specify master/slave interfaces for testing purposes. */ 
+#define SPI_TEST_MASTER SPI_2
+#define SPI_TEST_SLAVE SPI_0
+#define SPI_TEST_MASTER_PIN(SPI_PIN) SPI_2##_##SPI_PIN
+#define SPI_TEST_SLAVE_PIN(SPI_PIN)  SPI_0##_##SPI_PIN
```

Comment:
I can see the reason you want to adding these macros. I am OK with this.
Russ is working on Pinmaps, @c1728p9 do you have better suggestions?